### ISSUE TITLE
[BUG]: CSS or... What's going on?

### ISSUE THREAD

**Issue by Blaadick** [alexandersokol/sd-model-organizer#36]
What with or without the other plugins, on any theme, with any option to import the model. It does not work.

**OS**: win 11
**SD WebUi**: v1.4.0
**Browser**: Microsoft Edge

_Records list card_
![image](https://github.com/alexandersokol/sd-model-organizer/assets/123119046/1733a03c-e183-4942-8022-8b2160d29558)

_and any other tab of this plugin_

**Comment by alexandersokol** [comment]
Hello, I've just checked extension on the same environment and everything looks good. Do you have any other custom css styles for sd-webui or extensions?

**Comment by Blaadick** [comment]
Yes, I am using Lobe Theme. I assumed it was breaking the Model Organizer,
but after disabling Lobe Theme nothing has changed.

**Comment by alexandersokol** [comment]
Well, I tried both lobe theme and quick css, and it broke css styles for model organizer and few other extensions I have, and reverting changes to default has no effect. I will try to found out how it could be fixed.

**Comment by alexandersokol** [comment]
Fresh installation of Lobe theme and model orginizer works well. Did you use anything else f or theming?
<img width="1272" alt="image" src="https://github.com/alexandersokol/sd-model-organizer/assets/1095159/43ef75c6-e843-47a0-858e-dc0534bef0bd">

**Comment by Blaadick** [comment]
I only use Lobe Theme for WebUi design. I tried downloading WebUi and Model Organizer again, but it had no effect. Perhaps there is something wrong with something outside of WebUi. What information can I provide to better understand the problem?

**Comment by alexandersokol** [comment]
To be honest I'm not  sure that problem is in css styles. Your screenshot looks like they weren't loaded at all. If it is possible you can try to find `javascript/main.js` file in the extensions dir, find and uncomment the following line:
```
function log(text) {
    // console.log(text)
}
```
- Than save the file 
- launch sd-webui
- open it in browser
- open developer tools `ctrl+shift+i`
- open console tab
- reload the page
- open `Model Organizer` tab
- and send me what console output you have.

And can you also attach your webui start parameters if you use any?

**Comment by Blaadick** [comment]
Fixed it! The extension couldn't find the paths to some files, due to changing the local directory name during installation.

![Screenshot 2023-07-05 110902](https://github.com/alexandersokol/sd-model-organizer/assets/123119046/c965af5e-2aa4-4c1a-9b44-81f7ea28dc30)

**Comment by alexandersokol** [comment]
Great! Did you change dir path to extensions somewhere in the code or with argument?

**Comment by Blaadick** [comment]
I renamed the plugin folder in the `stable-diffusion-webui\extensions`

**Comment by Blaadick** [comment]
Only now I realize I've given little to no information. Well, when downloading the plugin, I set its folder to a custom name `ModelOrganizer`, but when loading, the plugin is looking for css files in the `sd-model-organizer` folder. Because of this, the error with css loading occurs.

**Comment by ceruleandeep** [comment]
@alexandersokol your method of using file= URLs to load content does not work if the extension is installed anywhere other than a subdirectory of `stable-diffusion-webui`. For instance, if user has relocated their data dir with `--data-dir
/disk2/sd/userdata` or sth, it will not work. Or as @Blaadick finds, it is not possible to rename the repository.

I tried to fix this, but I don't know how a1111 expects extensions to load content dynamically. Probably it expects that you will not! You can fix the css loading easily by moving it to the root dir of the extension, then a1111 will handle the loading for you. But I don't know how to fix tinymce loading. 

I gave up and cloned the repo into `stable-diffusion-webui/extensions`. This is the only extension I have that requires this.

Thank you for an excellent extension!
